release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -645,6 +668,52 @@ For more information, see xref:../openshift_images/image-configuration.adoc#imag
 
 You can use the new `APIRequestCount` API to track what is using the deprecated APIs. This allows you to plan whether any actions are required in order to upgrade to the next release.
 
+[id="ocp-4-8-monitoring-stack-dependency-version-updates"]
+==== Version updates to monitoring stack components and dependencies
+
+{product-title} 4.8 includes version updates to the following monitoring stack components and dependencies:
+
+* The Prometheus Operator is now on version 0.48.1.
+* Prometheus is now on version 2.26.1.
+* The `node-exporter` agent is now on version 1.1.2.
+* Thanos is now on version 0.20.2.
+* Grafana is now on version 7.5.5.
+
+[id="ocp-4-8-monitoring-kube-state-metrics-v2.0.0-upgrade"]
+==== kube-state-metrics upgraded to version 2.0.0
+
+`kube-state-metrics` is upgraded to version 2.0.0. The following metrics were deprecated in `kube-state-metrics` version 1.9 and are effectively removed in version 2.0.0:
+
+* Non-generic resource metrics for pods:
+** kube_pod_container_resource_requests_cpu_cores
+** kube_pod_container_resource_limits_cpu_cores
+** kube_pod_container_resource_requests_memory_bytes
+** kube_pod_container_resource_limits_memory_bytes
+* Non-generic resource metrics for nodes:
+** kube_node_status_capacity_pods
+** kube_node_status_capacity_cpu_cores
+** kube_node_status_capacity_memory_bytes
+** kube_node_status_allocatable_pods
+** kube_node_status_allocatable_cpu_cores
+** kube_node_status_allocatable_memory_bytes
+
+[id="ocp-4-8-monitoring-removed-grafana-alertmanager-ui-links"]
+==== Removed Grafana and Alertmanager UI links
+
+The link to the third-party Alertmanager UI is removed from the *Monitoring* -> *Alerting* page in the {product-title} web console. Additionally, the link to the third-party Grafana UI is removed from the *Monitoring* -> *Dashboards* page. You can still access the routes to the Grafana and Alertmanager UIs in the web console in the *Administrator* perspective by navigating to the *Networking* -> *Routes* page in the `openshift-monitoring` project.
+
+[id="ocp-4-8-monitoring-dashboards-updates-web-console"]
+==== Monitoring dashboard enhancements in the web console
+
+New enhancements are available on the *Monitoring* -> *Dashboards* page in the {product-title} web console:
+
+* When you zoom in on a single graph by selecting an area with the mouse, all other graphs now update to reflect the same time range.
+* Dashboard panels are now organized into groups, which you can expand and collapse.
+* Single-value panels now support changing color depending on their value.
+* Dashboard labels now display in the *Dashboard* drop-down list.
+* You can now specify a custom time range for a dashboard by selecting *Custom time range* in the *Time Range* drop-down list.
```

Comment:
Small wording update to match the Monitoring docs: s/drop-down menu/drop-down list/